sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -42,6 +42,11 @@
                         string="Cancel"
                         class="btn-secondary"
                         type="object"
+                        attrs="{'invisible': [
+                            '|',
+                            ('state', 'not in', ['draft', 'open']),
+                            ('id', '=', False)
```

Comment:
**chore(non-blocking)**: isn't the second condition `('id', '=', False)` redundant? When a new SBO is created, it's in `draft` state. I think this condition could be simplified by only leaving the first tuple.

Reply:
The point was to hide the Cancel button when creating a new record, because it doesn't exist yet so we cannot cancel it

Reply:
For sure. But a new record is also in 'draft', so the button should already be hidden by the first condition, no?

Anyway, this is a non-blocking comment, so feel free to ignore it.

Reply:
It was done this way to mimic as close as possible the way `sale` module works.

See how it was done in the form view for `sale.order`

https://github.com/odoo/odoo/blob/c77881763c11871dc061c47eefcca707b24e8a4e/addons/sale/views/sale_views.xml#L290